packages/alfa-list/src/list.ts

```diff
@@ -90,6 +90,10 @@ export class List<T> implements Collection.Indexed<T> {
     );
   }
 
+  public flatten<T>(this: List<List<T>>): List<T> {
+    return this.flatMap((list) => list);
+  }
```

Comment:
Could it be 
```suggestion
  public flatten<T>(this: List<Iterable<T>>): List<T> {
    return this.flatMap((iterable) => List.from(iterable));
  }
```

Reply:
That breaks the signature of a monadic join: `F<F<T>> => F<T>`.

Reply:
I should actually add the method to the `Monad<T>` interface.